js/field.js

```diff
@@ -716,14 +716,18 @@ $(function() {
                 lookup: function(query, done) {
                     var values = BUGZILLA.autocomplete_values[that.data('values')];
                     query = query.toLowerCase();
+                    var activeValues = that.val().split(',');
```

Comment:
This line needs to be changed as currently it will only read in the first time and not get any updates to the field if someone removes or adds a keyword. Change to:

var activeValues = $('#keywords').val().split(',');

This will cause it to re-read each time the current set of field values.

Reply:
Should I use document.querySelector instead of '$' to make migration easier as Emma suggested?